src/components/editor/markdown-renderer/replace-components/image/image-frame.tsx

```diff
@@ -1,7 +1,26 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import { useSelector } from 'react-redux'
+import { getProxiedUrl } from '../../../../../api/imageProxy'
+import { ApplicationState } from '../../../../../redux'
+
+export const ImageFrame: React.FC<React.ImgHTMLAttributes<HTMLImageElement>> = ({ alt, src, ...props }) => {
```

Comment:
What still bugs me about this component is, that it needs two renderings for a non-proxied image. How about rendering the img tag with the src, if the proxy is not enabled?

Reply:
Changed in 581d7028